msteams-platform/concepts/authentication/nested-authentication.md

```diff
@@ -0,0 +1,343 @@
+---
+title: SSO authentication for nested apps
+description: Learn how to implement, configure nested app authentication in Microsoft Teams app. Learn about the use case scenarios for nested app authentication.
+ms.date:
+ms.topic: conceptual
+author: v-ypalikila
+ms.author: surbhigupta
+ms.localizationpriority: medium
+---
+
+# Nested app authentication
+
+> [!NOTE]
+>
+> * Nested app authentication (NAA) is available only in [public developer preview](../../resources/dev-preview/developer-preview-intro.md).
+> * NAA is supported only in single-page application (SPA), such as tabs.
+
+NAA is a new authentication protocol for SPA that are embedded in host environments, such as Teams, Outlook, and Microsoft 365. It simplifies the authentication process to facilitate single sign-on (SSO) across apps nested within supported host apps. The NAA model supports a primary identity for the host app that includes multiple app identities for nested apps. Microsoft utilizes this model in Teams tabs, personal apps, and Office Add-ins.
+
+The NAA model provides several advantages over the On-Behalf-Of (OBO) flow:
```

Comment:
```suggestion
The NAA model provides several advantages over the On-Behalf-Of (OBO) flow:
```
```suggestion
The NAA model provides several advantages over the [On-Behalf-Of (OBO)](/entra/identity-platform/v2-oauth2-on-behalf-of-flow) flow:
```

Should we link over to identity docs here?

Not sure if the relative path will work here, but trying to link this doc while keeping localization intact

https://learn.microsoft.com/en-us/entra/identity-platform/v2-oauth2-on-behalf-of-flow

Reply:
@iambmelt I had refrained from adding a link within content here to keep the reader from getting distracted. I've added it to the See Also section.